fog/ingest/server/src/lib.rs

```diff
@@ -36,7 +36,7 @@ use mc_util_metrics::ServiceMetrics;
 
 lazy_static::lazy_static! {
     /// Generates service metrics for tracking
-    pub static ref SVC_COUNTERS: ServiceMetrics = ServiceMetrics::new_and_registered("fog_ingest");
+    pub static ref SVC_COUNTERS: ServiceMetrics = ServiceMetrics::new_and_registered("fog_ingest_service");
```

Comment:
:question: Making sure I understand this change. The previous name "fog_ingest" name collided with `fog/ingest/server/src/counters.rs`
```rust
    pub static ref OP_COUNTERS: OpMetrics = OpMetrics::new_and_registered("fog_ingest");
```
resulting in the same counter being used for both and thus both metrics in the `fog_ingest` namespace.

Reply:
When the name collided the OpMetrics would show up but not the ServiceMetrics